markdown/org/docs/sewing/knit-binding/en.md

```diff
@@ -79,41 +80,50 @@ Stretch your binding too much, and it will spring back too far, causing the main
 
 ![As the curve changes, so should the stretch](knitbinding3.png)
 
-Straight seams exist, and although I've never come across one, maybe uniformly curved seams exist too. But more often than not, the curvature of your seam will vary.
+Straight seams exist, and perhaps uniformly curved seams exist too. But more often than not, the curvature of your seam will vary.
 
 On a typical T-shirt neckline, the curve at the back of your neck is different from the front, and there's probably a somewhat tighter bend behind your shoulder seam.
 
-On a slight curve, the difference between the inner and outer edge is not so big, and a little bit of stretch is all you need. But on a tighter curve, the difference between the inner and outer edge of your binding will be more significant. And you'll need more stretch to accommodate for that.
+On a slight curve, the difference between the inner and outer edge is not so big, and a little bit of stretch is all you need. But on a tighter curve, the difference between the inner and outer edge of your binding will be more significant. And, you'll need more stretch to accommodate for that.
 
-So as you follow your seam, and it curves around necklines, or shoulders, or design features, you will have to adapt the amount of stretch to the situation.
+So as you follow your seam and as it curves around necklines, shoulders, or design features, you will have to adapt the amount of stretch to the situation.
 
 ## You cannot calculate the length of your knit binding
 
-I am not saying it is theoretically impossible. But I am saying that it is practically impossible.
+It may not be theoretically impossible, but it is practically impossible.
```

Comment:
I am concerned about the fact that this change by removing personal language suggests this is always true. I don't think that is the case.

Reply:
I've sewed 2 knit bands so far. What I do, after the pattern is printed and cut, is to use thread to measure the opening, then measure the thread length. I 'walk it off' on the opening using thread. 
Maybe this 'trick' can be written down in this document as well?

What this refers to IMO, is that it may not be possible or feasible to get that measurement in the digital version of the pattern. 

I'm not saying to not discuss it but personally I prefer documentation to be impersonal.

Reply:
I think the reason there's no author stated is that this was written before Freesewing was on GitHub, hence my assumption Joost wrote this. 

We can definitely have a discussion in a contributor call about how we want the freesewing documentation around sewing to sound. 

I personally do my best to not change the original writing or tone when I change or review documentation unless there is an issue around clarity or some kind of error.